### PR TITLE
#29 Connect Menu

### PR DESCRIPTION
**Pull request by AgaMazur**
- connect menu scenes

- on click 'graj' go to 'podaj imię' scene
- on click 'dalej' go to 'menu' scene
- on click 'rybki' go to 'fishgame' scene

### CODE REVIEW

**Review by przemytka — DISMISSED** [review]

**Review by juliakobusinska98 — DISMISSED** [review]

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/GetPlayerName.cs [hunk 1]
```diff
@@ -12,5 +13,7 @@ public void Button_Click()
     {
         playerName = inputField.GetComponent<Text>().text;
         Debug.Log( "Witaj " + playerName + " w podwodnym świecie tabliczki mnożenia!");
+        SceneManager.LoadScene("Menu");
+
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/GetPlayerName.cs]: unnecessary empty line

**Review by ani119 — CHANGES_REQUESTED** [review]
It is almost ready. I only ask to delete one unnecessary empty line :)

**Review by juliakobusinska98 — APPROVED** [review]

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/GetPlayerName.cs (hunk 1 above)
Comment (AgaMazur) [reply podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/GetPlayerName.cs]: changed in second commit 81b84f2

**Comment by ani119** [comment]
**Outdated:** I am not sure but meseems that there is a problem with main camera at game scene - it is too close and i don't see all fish. Before update i was only look at code now i see it unity project

image

**Review by ani119 — APPROVED** [review]

**Review by przemytka — APPROVED** [review]

**Pull request merged**